src/main/drivers/rx/rx_spi.c

```diff
@@ -47,11 +49,33 @@
 static busDevice_t rxSpiDevice;
 static busDevice_t *busdev = &rxSpiDevice;
 
+static IO_t extiPin = IO_NONE;
+static extiCallbackRec_t rxSpiExtiCallbackRec;
+static bool extiLevel = true;
+
+static volatile bool extiHasOccurred = false;
+static volatile timeUs_t lastExtiTimeUs = 0;
+
+static timeDelta_t lastFrameDeltaUs = 0;
+
 void rxSpiDevicePreInit(const rxSpiConfig_t *rxSpiConfig)
 {
     spiPreinitRegister(rxSpiConfig->csnTag, IOCFG_IPU, 1);
 }
 
+void rxSpiExtiHandler(extiCallbackRec_t* callback)
+{
+    UNUSED(callback);
+
+    const timeUs_t extiTimeUs = microsISR();
+
+    if (IORead(extiPin) == extiLevel) {
+        lastFrameDeltaUs = cmpTimeUs(extiTimeUs, lastExtiTimeUs);
```

Comment:
lastFrameDeltaUs is not `volatile`, it may cause problems if something is checking it in quite tight loop (which isn't probably the case).
There will also be race with clearing function (volatile won't solve it), but that's also mostly theoretical